plugin.video.vtm.go/CHANGELOG.md

```diff
@@ -0,0 +1,221 @@
+# Changelog
```

Comment:
In previous reviews the Kodi devs have requested to *not* include changelogs and use the addon.xml news section for including recent changelog information.

Reply:
I've removed the `CHANGELOG.md` file.